py/kubeflow/kfctl/testing/pytests/kf_is_ready_test.py

```diff
@@ -101,6 +101,8 @@ def test_kf_is_ready(record_xml_attribute, namespace, use_basic_auth, use_istio,
       "centraldashboard",
       "jupyter-web-app-deployment",
       "minio",
+      "metadata-deployment",
```

Comment:
Instead of adding it to the existing test can we create a separate function for it? Similar to test_katib_is_ready.
I think we want to have many small tests as opposed to 1 monolithic test. This makes it easier to deal with flakes;
e.g. if one particular app is flaky we mark that specific test as expected to fail.

In spyglass and testgrid we also know which application is having problems as opposed to just kf isn't ready.

Reply:
Good suggestion! Done.